ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -143,3 +146,15 @@ def _process_step_infos(self, step_infos: List[EnvironmentStep]) -> int:
                     step_info.environment_stats, step_info.worker_id
                 )
         return len(step_infos)
+
+    @staticmethod
```

Comment:
bump

Reply:
I made it a static method on the env manager since only the env_managers use it. We don't have a general utils file though which may be a better place for this to live

Reply:
@vincentpierre this is removed in the ActionModel